packages/grid/x-data-grid-premium/src/components/GridColumnMenuAggregationItem.tsx

```diff
@@ -18,6 +18,22 @@ import {
 import { gridAggregationModelSelector } from '../hooks/features/aggregation/gridAggregationSelectors';
 import { GridAggregationModel } from '../hooks/features/aggregation/gridAggregationInterfaces';
 
+const MenuItem = styled(MUIMenuItem)(() => ({
+  '&:hover': {
```

Comment:
The cursor feels strange, it feels me that I can click but clicking doesn't do anything:

https://github.com/mui/mui-x/assets/3165635/ab02a25c-8cd2-4a21-abb0-4a1d742e96d6

So either:

```suggestion
  cursor: 'default',
  '&:hover': {
```

or we wire the click to focus the select and open it.

Reply:
Yes, that could be a useful mode to handle such use-cases.